### DISCUSSION TITLE
p-table custom cell renderer

### DISCUSSION THREAD

**Discussion by MahmoudTarek44** [primefaces/community#93] [PrimeNG]
I was wondering is it available or not to render a custom cell dynamically in the table as I predefine the row configuration for each cell to create a full custom table row
Creating a separate reusable cell components and just pass it maybe link 
tableConfig = [
  {
      Name: 'Name',
      Component: NameComponent
      Other configurations: ....
   },
  {
      Name: 'Age',
      Component: AgeComponent
      Other configurations: ....
   }
]
I still don't know in this approach how the row data will be passed into the cell..
And with no component selected it is the default cell will be rendered.
Is this kind of configuration applicable or other similar approach?

**Comment by cetincakiroglu** [comment]
Hi,

It's difficult to understand what are you trying to do, could you please share a StackBlitz example so we can identify the issue?
